apps/wallet/src/ui/app/pages/home/transactions/index.tsx

```diff
@@ -1,19 +1,71 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
-import { memo } from 'react';
 
-import PageTitle from '_app/shared/page-title';
-import RecentTransactions from '_components/transactions-card/RecentTransactions';
+import { useQueryClient } from '@tanstack/react-query';
+import { memo, useEffect } from 'react';
 
-import st from './Transactions.module.scss';
+import PageTitle from '_app/shared/page-title';
+import { ErrorBoundary } from '_components/error-boundary';
+import Loading from '_components/loading';
+import { Transaction } from '_components/transactions-card/Transaction';
+import { useAppSelector, useGetTransactionsByAddress } from '_hooks';
+import Alert from '_src/ui/app/components/alert';
 
 function TransactionsPage() {
+    const activeAddress = useAppSelector(({ account: { address } }) => address);
+    const selectedApiEnv = useAppSelector(({ app }) => app.apiEnv);
+    const queryClient = useQueryClient();
+
+    // Switching between accounts should invalidate the query cache
```

Comment:
This seems unnecessary, just leave it in cache and let it get evicted at some point.

Reply:
removing this